sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -174,8 +173,7 @@ object Klaviyo {
      * @return Returns [Klaviyo] for call chaining
      */
     fun setProfile(profile: Profile): Klaviyo = apply {
```

Comment:
Note to self: check if iOS implementation is chainable.

Reply:
arguably this one doesn't really need it, but i could see maybe wanting to do 
```
Klaviyo.setProfile(profile)
   .setPushToken(token)
```